### PR TITLE
Update upgrade script to copy config

### PR DESCRIPTION
**Pull request by NehaNaithani**

### CODE REVIEW

File: packaging/iofog-agent/usr/share/iofog-agent/upgrade.sh [hunk 1]
```diff
@@ -51,6 +53,11 @@ get_distribution() {
 	esac
 
 	# Restore config and start agent
+	cd /var/backups/iofog-agent
+	tar -xzf config_backup$iofogpackage.tar.gz
+	mv etc/iofog-agent/* /etc/iofog-agent/
+	printf 'config restored'
```
Comment (sergerad) [inline packaging/iofog-agent/usr/share/iofog-agent/upgrade.sh]:
```
echo 'config restored'
```
for newline

Comment (NehaNaithani) [reply packaging/iofog-agent/usr/share/iofog-agent/upgrade.sh]: updated

File: packaging/iofog-agent/usr/share/iofog-agent/upgrade.sh [hunk 2]
```diff
@@ -51,6 +53,11 @@ get_distribution() {
 	esac
 
 	# Restore config and start agent
+	cd /var/backups/iofog-agent
+	tar -xzf config_backup$iofogpackage.tar.gz
+	mv etc/iofog-agent/* /etc/iofog-agent/
```
Comment (baghbidi) [inline packaging/iofog-agent/usr/share/iofog-agent/upgrade.sh:58]: Shouldn't this be `mv /var/backups/iofog-agent/* /etc/iofog-agent/`?

Comment (NehaNaithani) [reply packaging/iofog-agent/usr/share/iofog-agent/upgrade.sh:58]: It's already in the backup directory `cd /var/backups/iofog-agent` and when we untar the files are created in the `/var/backups/iofog-agent/etc/iofog-agent/` directory. 
That's why `mv etc/iofog-agent/* /etc/iofog-agent/`.

**Review by baghbidi — APPROVED** [review]

**Pull request merged**